pkg/templates/csi.go

```diff
@@ -111,7 +111,8 @@ var CSIOperatorConfigSpec = csiopv1a1.OperatorConfigSpec{
 					},
 				},
 			},
-			Replicas: ptr.To(int32(2)),
+			Replicas:    ptr.To(int32(2)),
+			HostNetwork: ptr.To(true),
```

Comment:
Why this is set to true here if its controlled in the operatorConfig CR?

Reply:
@Madhu-1 
We only have automation control over the config map for the internal client (via ocs-operator). Not for any other client. This means we must assume `true` as a default value then have ocs-opeartor turn this off for "internal mode" like deployments.

In any case, this is a temporary workaround and we do have a better long term solution coming soon

Reply:
what will happen for the mode where we have no internal but only external ceph cluster?

Reply:
@Madhu-1 What are you expecting it to be in that case? with or without setting the controller to use host network?
Please remember that the only reason we are setting the controller to use host network is when we apply data separation technics without Multus. I don't think we support it with external mode only deployments

Reply:
For external mode, we should still be using pod networking only (this is the current behavior today)